src/config.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
   baseURL: process.env.baseURL || 'http://localhost:8080',
   jwtSigningKey: new Buffer(jwtSigningKeyB64, 'base64'),
   sqlite: process.env.sqlite || './data/podverseWeb.sqlite',
-  databaseName: 'podverse'
+  databaseName: 'podverse',
+
+  // TODO: change the auth0 values to podverse.fm's clientId and domain before deployment
+  auth0ClientId: 'tTGQrl5CenMDdpzcKNpmLIginyRgBJNN',
```

Comment:
whatever could be a 'default' that is non-sensitive.  when deployed we would invoke

    export auth0ClientId='the-real-client-id`
    npm start